lib/meadow/data/action_states.ex

```diff
@@ -30,7 +30,7 @@ defmodule Meadow.Data.ActionStates do
   end
 
   def error?(object_id, action), do: latest_outcome(object_id, action) == "error"
-  def ok?(object_id, action), do: latest_outcome(object_id, action) == "ok"
+  def ok?(object_id, action), do: Enum.member?(["ok", "n/a"], latest_outcome(object_id, action))
```

Comment:
I'd like to see this list put in an `@ok_states` module variable or something to avoid repetition (and future search/replace refactor errors)